pyproject.toml

```diff
@@ -33,10 +33,11 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.8"
-faker = {version = "~=30.0", optional = true}
-psycopg2-binary = "2.9.9"
+faker = {version = "~=29.0", optional = true}
 sqlalchemy = "~=2.0"
 sshtunnel = "0.4.0"
+psycopg = "^3.2.3"
+psycopg-binary = "^3.2.3"
```

Comment:
Is there a reason we both the source and binary packages?

Reply:
@edgarrmondragon This can be changed to just psycopg. Although now I am wondering if a user wants to use psycopg[c] or psycopg[binary] what would be the suggestion?

Reply:
https://github.com/dbt-labs/dbt-postgres/issues/96 is probably a good case study. Most users in the data space can't or don't want to build C extensions, so we'll probably prefer `psycopg[binary]`.